.ci/upload.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+rm -rf dist
+mkdir dist/
+
+export STORAGE_ACCOUNT_NAME=platopslackhelpbotai
+export CONTAINER_NAME=the-hmcts-way
+
+function extractMain () {
+    local path=$1
+
+    DIRECTORY=$(dirname "$path")
+    mkdir -p dist/$DIRECTORY
+
+    cat $path | htmlq --text 'main' --remove-nodes '[data-module="page-expiry"]' > dist/$path
+}
+
+export -f extractMain
+
+find build -name "*.html" -not -path "build/search/*" -exec bash -c "extractMain \"{}\"" \;
+
+# we use sync here so that deleted files get removed
+azcopy sync --compare-hash=md5 \
+  --delete-destination=true \
+  --include-pattern="*.html"  \
+  "dist/" "https://$STORAGE_ACCOUNT_NAME.blob.core.windows.net/$CONTAINER_NAME/"
+
+function setTitle() {
```

Comment:
bit of a pain having to do this but not sure of a way to do it with sync, it works, just takes a bit of time